CHANGES.rst

```diff
@@ -18,6 +18,54 @@
 Changes
 =======
 
+`1.2.3 <https://github.com/SwissDataScienceCenter/renku-python/compare/v1.2.2...v1.2.3>`__ (2022-04-29)
+-------------------------------------------------------------------------------------------------------
+
+Bug Fixes
+~~~~~~~~~
+
+-  **cli:** inform user if a dataset wasn’t found in dataset show
+   (`#2830 <https://github.com/SwissDataScienceCenter/renku-python/issues/2830>`__)
+   (`046a756 <https://github.com/SwissDataScienceCenter/renku-python/commit/046a7562bb885129058c1e523594785de804d2ca>`__)
+-  **core:** tests for renku session subcommands
+   (`#2814 <https://github.com/SwissDataScienceCenter/renku-python/issues/2814>`__)
+   (`a1a07c7 <https://github.com/SwissDataScienceCenter/renku-python/commit/a1a07c766d674c53ecf37a4c3338133c86edbd06>`__)
+-  **dataset:** correct dataset image id after migration to v1.0.0
+   (`#2842 <https://github.com/SwissDataScienceCenter/renku-python/issues/2842>`__)
+   (`c2e08c8 <https://github.com/SwissDataScienceCenter/renku-python/commit/c2e08c8afeacf09bcd3f541ad7d48fb4ec72b929>`__)
+-  **service:** fix project_id not being autogenerated if missing in
+   request schema
+   (`#2828 <https://github.com/SwissDataScienceCenter/renku-python/issues/2828>`__)
+   (`ab46cd5 <https://github.com/SwissDataScienceCenter/renku-python/commit/ab46cd5672e96865997c0f53d8dc59e7cb0ccb09>`__)
+
+Features
+~~~~~~~~
+
+-  **cli:** gc command
```

Comment:
Might want to write something more descriptive, like "added renku gc command for cleaning up renku cache"